lib/src/executables/intl_message_migration.dart

```diff
@@ -292,9 +293,14 @@ Iterable<String> dartFilesToMigrate() => Glob('**.dart', recursive: true)
 
 Iterable<String> dartFilesToMigrateForPackage(
         String package, Set<String> processedPackages) =>
-    Glob(p.join(package, 'lib', '*', '**.dart'), recursive: true)
-        .listSync()
+    // Glob is peculiar about how it wants absolute Windows paths, so just query the
+    // file system directly. It wants "posix-style", but no leading slash. So
+    // C:/users/user/..., which is ugly to produce.
+    fs
+        .directory(p.join(package, 'lib'))
```

Comment:
I think the glob was excluding files directly in /lib, which seemed wrong. They might not often have messages, but they might.

Reply:
should we also be excluding the <package>_intl.dart file itself?

Reply:
We should. I was doing that in a separate PR for making the codemod able to run repeatedly. But it's a one-liner, it would be easy to add here.

Reply:
ok, want to go ahead and add a 
`.where((file) => !file.path.endsWith('_test.dart'))`

Reply:
Done.